Chisel/Makefile

```diff
@@ -0,0 +1,11 @@
+PREFIX ?= /usr/local/lib
+
+install:
+	xcodebuild \
+		-scheme Chisel \
+		-configuration Release \
+		-sdk iphonesimulator \
+		install \
+		DSTROOT="$(PREFIX)" \
+		INSTALL_PATH=/ \
+		SKIP_INSTALL=NO
```

Comment:
Should this be set to `NO` in the `.xcodeproj`? I'm fine with it here either way, but I'm wondering why not have it default to this anyway.

Reply:
I'll set it in the project since there's no reason to archive it anyways